$OpenBSD: patch-lib_en_refman_index_so,v 1.1 2011/03/22 15:27:57 sthen Exp $
# Make the table a bit narrower, such that the build doesn't fail with:
# pic:lib/en/refman/i18n.pic.so:23: giving up on this picture
# warning: file `lib/en/refman/index.so', around line 4:
#   table wider than line width
--- lib/en/refman/index.so.orig	Mon Aug 26 06:29:53 2002
+++ lib/en/refman/index.so	Thu Mar 17 17:11:53 2011
@@ -1,6 +1,6 @@
 .hy 0
 .TS
-l r lw(2.5i) lw(2.5i).
+l r lw(2.0i) lw(2.0i).
 cook_rsh(1)	\*[Page*Of*cook_rsh(1)]	T{
 .ad r
 cook_rsh - load
